lib/utf8.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports.decode = function (input) {
+  if (Buffer.isBuffer(input) === false) {
```

Comment:
Compare [line 115](https://github.com/jsdom/whatwg-url/pull/77/files/4f29550994ed489dcdf8f8fd774d7c820c04ed45#diff-edf3f529109e5fb5467bfde0c58cd585L115) to [line 121](https://github.com/jsdom/whatwg-url/pull/77/files/4f29550994ed489dcdf8f8fd774d7c820c04ed45#diff-edf3f529109e5fb5467bfde0c58cd585L121)